TransactionMobile/TransactionMobile/Views/MobileTopup/MobileTopupSelectProductPage.xaml.cs

```diff
@@ -0,0 +1,116 @@
+﻿namespace TransactionMobile.Views.MobileTopup
+{
+    using System;
+    using System.Diagnostics.CodeAnalysis;
+    using Events;
+    using Models;
+    using Pages;
+    using Syncfusion.XForms.Buttons;
+    using ViewModels;
+    using Xamarin.Forms;
+    using Xamarin.Forms.Xaml;
+
+    /// <summary>
+    /// 
+    /// </summary>
+    /// <seealso cref="Xamarin.Forms.ContentPage" />
+    /// <seealso cref="TransactionMobile.Pages.IMobileTopupSelectProductPage" />
+    /// <seealso cref="TransactionMobile.Pages.IPage" />
```

Comment:
Codacy found an issue: ['partial' is gratuitous in this context.](https://app.codacy.com/manual/stuart_ferguson1/MobileApplication/pullRequest?prid=5918284)